indico/migrations/versions/201705221530_3ca338ed5192_remove_background_image_and_add.py

```diff
@@ -22,9 +22,12 @@ def upgrade():
                           'designer_templates', 'designer_templates',
                           ['backside_template_id'], ['id'],
                           source_schema='indico', referent_schema='indico')
+    op.create_index('ix_designer_templates_backside_template_id', 'designer_templates', ['backside_template_id'],
```

Comment:
Well, I checked in our codebase for `op.create_index(` and it seems like we don't have a pattern. But sure, I can use `None`!